samples/batch/python/README.md

```diff
@@ -1 +1,15 @@
 Examples in C# and Python on how to use the Speech Services Batch Transcription API.
+
+To execute the sample you need to generate the Python client library.
+
+To do this following the steps:
+
+1. Go to https://editor.swagger.io
+2. Click File, then click Import
+3. Enter the swagger URL including the region for your Speech Services subscription https://<your-region>.cris.ai/docs/v2.0/swagger
+4. Click Generate Client and select Python
+5. Save the client library
+
+Copy the library in the folder python-library.
```

Comment:
Extract as well?
Should it be in its own directory?
If not, what about the files that are in the generated .zips and the files that are already in here?
For examples, requirements.txt is in the generated .zip, but there's one in the samples directory already - should it be replaced? or not?

Reply:
no it should not be on its own directory. the files there remain intact. this is about copying the entire folder (client library) into the python sample project you download.

Reply:
I think it would be better to pip-install the python client:

6. Extract the downloaded `python-client-generated.zip` somewhere in your file system.
7. Install the extracted `python-client` module in your python environment using pip: `pip install path/to/package/python-client`.
8. The installed package has the name `swagger_client`. You can check that the installation worked using the command `python -c "swagger_client".

Reply:
With these changes, the files requirements.txt and setup.py can be deleted. main.py should be on the same level as README.md, or in a separate directory that isn't called python-client, to not create confusion with the downloaded package.